pkg/server/router/credential.go

```diff
@@ -145,6 +147,140 @@ func (cr CredentialRouter) GetCredential(ctx context.Context, w http.ResponseWri
 	return framework.Respond(ctx, w, resp, http.StatusOK)
 }
 
+type GetCredentialStatusResponse struct {
+	Revoked bool `json:"revoked"`
```

Comment:
I remember having a conversation around `suspendable` credentials. What was the final decision on that?

Reply:
aligns with my comment above. we can choose to support specific statuses like 'revoked' or 'suspended' or leave it open to the administrators to define the statuses they want to have present on each credential - which could be none, one, or many

Reply:
I'll expand this status object to have revoked and suspended. 
In the future if there is a need to have a lot more status we can change it to be open eneded.

Reply:
sounds good perhaps a new issue that you can reference in code that covers this

Reply:
Added new issue that adds suspended in the future - https://github.com/TBD54566975/ssi-service/issues/177